### PR TITLE
Filter blocks from toolbar config

### PR DESCRIPTION
**Pull request by andrew-w**
Added ability to filter from the toolbar config page.

### CODE REVIEW

**Comment by lord-devious** [comment]
Me like. :trophy:

File: Sources/Sandbox.Game/Game/Entities/Blocks/MyBeacon.cs [hunk 1]
```diff
@@ -53,6 +53,11 @@ static MyBeacon()
             show.Setter = (x, v) => x.RequestShowInTerminal(v);
             MyTerminalControlFactory.AddControl(show);
 
+            show = new MyTerminalControlOnOffSwitch<MyBeacon>("ShowInToolbarConfig", MySpaceTexts.Terminal_ShowInToolbarConfig, MySpaceTexts.Terminal_ShowInToolbarConfigToolTip);
```
Comment (RossM) [inline Sources/Sandbox.Game/Game/Entities/Blocks/MyBeacon.cs]: Why add this here (and in MyRadioAntenna)? The control should already be added by MyTerminalBlock.

**Comment by lord-devious** [comment]
Probably because this is how Keen did it with their ShowInTerminal?

**Comment by mexmer** [comment]
just note, always create new var for your control. you did that in terminalblock, but for some reason you reused "show" variable in others ....

using new variable is better to prevent errors, in case someone rely on that value in later code ... i know it's short, but still ... someone might later add aditional logic to controls at end of fucntions, and here you go ....

**Comment by OndrejPetrzilka** [comment]
Can someone explain me purpose of this? This is to remove mess in G screen when putting things on ship toolbar?

Please use new variable instead of 'show' as someone suggested.

**Comment by mexmer** [comment]
@OndrejPetrzilka yes, seems to be same kind of filter like "Show in terminal", but this time for toolbar (g menu)

**Comment by andrew-w** [comment]
Updated code as requested to use new vars instead of re-using old ones.

And yes, this is to hide blocks in the ship toolbar G screen.  To reduce the mass of blocks listed when configuring cockpit toolbars, timer blocks etc.

I have one question about the code though.  In the MySyncBlockHelpers.cs file, I used a number for the message ID (see line 80) which was close to the ShowInTerminalMsg ID.  Is there a proper way to determining which ID's to use?  or is just finding one that isn't in use good enough?  As that 's what I did for this one.

**Comment by OndrejPetrzilka** [comment]
There's no better way of determining message id now. Please update you branch so it can be automatically merged.

**Comment by andrew-w** [comment]
Updated branch  :)

**Pull request merged**